Sources/Commands/Workspace.swift

```diff
@@ -816,12 +832,6 @@ public class Workspace {
         // Inform the delegate.
         delegate.removing(repository: dependency.repository.url)
 
-        // If this specifier is pinned, emit a note about stray pin.
-        if let pin = pinsStore.pins.first(where: { $0.repository == specifier }) {
-            // FIXME: Use diagnosics engine when we have that.
-            print("note: Considering unpinning \(pin.package), it is pinned at \(pin.version) but the package is being removed.")
-        }
-
```

Comment:
This is moved above in `repinPackages(with:)` method where it is handled better. Now swiftpm will always emit this when repinning, having it here means it only emits once when repository is being removed.